mmv1/templates/terraform/examples/firebase_android_app_basic.tf.erb

```diff
@@ -0,0 +1,20 @@
+resource "google_project" "default" {
+	provider = google-beta
+
+	project_id = "tf-test%{random_suffix}"
+	name       = "tf-test%{random_suffix}"
+	org_id     = "<%= ctx[:test_env_vars]['org_id'] %>"
+}
+
+resource "google_firebase_project" "default" {
+	provider = google-beta
+	project  = google_project.default.project_id
+}
+
+resource "google_firebase_android_app" "<%= ctx[:primary_resource_id] %>" {
+	provider = google-beta
+	project = google_project.default.project_id
```

Comment:
nit: I'd put google_firebase_project.default.project here instead. that makes it abundantly clear the relationship between them.

Reply:
Project creation is skipped per the comment above.